src/xenia/gpu/gl4/command_processor.cc

```diff
@@ -2187,8 +2186,9 @@ CommandProcessor::UpdateStatus CommandProcessor::UpdateDepthStencilState() {
 
   bool dirty = false;
   dirty |= SetShadowRegister(regs.rb_depthcontrol, XE_GPU_REG_RB_DEPTHCONTROL);
-  dirty |=
-      SetShadowRegister(regs.rb_stencilrefmask, XE_GPU_REG_RB_STENCILREFMASK);
+  dirty |= SetShadowRegister(regs.rb_stencilrefmask, XE_GPU_REG_RB_STENCILREFMASK);
+  dirty |= SetShadowRegister(regs.rb_stencilrefmaskbf, XE_GPU_REG_RB_STENCILREFMASK_BF);
```

Comment:
did you spot it being the case that that this is true? I seem to remember the values being packed in the single stencil mask register